src/data/games/1836.json

```diff
@@ -1,19 +1,19 @@
 {
   "info": {
     "title": "1836",
-    "designer": "John Bohrer",
+    "designer": "Eddie Robbins",
     "background": "gray",
     "orientation": "horizontal",
     "titleX": 500,
-    "currency": "USD"
+    "currency": "$#"
```

Comment:
Why was this needed? Normally USD works (at least for me), was it not for you?

Reply:
It did not look good on the player table when I put it on the map (Firefox):
![afbeelding](https://user-images.githubusercontent.com/2109937/73336796-150bf980-4273-11ea-822f-872b4f771b62.png)